app/graphql/theme-setup.tsx

```diff
@@ -0,0 +1,40 @@
+import { useThemeMode } from "@rneui/themed"
+import React from "react"
+import { Appearance } from "react-native"
+import { useDarkModeQuery } from "./generated"
+
+export const ThemeSync = () => {
+  const { mode, setMode } = useThemeMode()
+  const { data } = useDarkModeQuery()
+  const darkMode = data?.darkMode
```

Comment:
Shoudn't this component just the `useDarkMode` hook and whenever the useDarkMode hook is different than the useThemeMode value, it should set the theme. The logic is duplicated somewhat from that hook and it adds complexity and introduces the possibility of the useDarkMode hook getting out of sync with the theme provider.

Reply:
I'm thinking to remove this code (everything related to graphql and darkmode) when it goes to prod. and solely rely on the OS settings. 

but for now maybe we can keep it while we're in development, to make it easier to switch between theme (without having to go to the OS settings, which also changes all the other apps design... quite cumbersome)